tests/sles4sap/qesapdeployment/deploy.pm

```diff
@@ -16,6 +16,19 @@ sub run {
     die "'qesap.py terraform' return: $ret[0]" if ($ret[0]);
     my $inventory = qesap_get_inventory(provider => get_required_var('PUBLIC_CLOUD_PROVIDER'));
     upload_logs($inventory, failok => 1);
+
+    # set up azure native fencing
+    if (get_var('QESAPDEPLOY_FENCING') eq 'native' && get_var('PUBLIC_CLOUD_PROVIDER') eq 'AZURE') {
```

Comment:
Why not?
```
if (check_var('QESAPDEPLOY_FENCING', 'native') && is_azure) {
```

P.S.: for `is_azure` you'll need to add `use publiccloud/utils qw(is_azure);`

Reply:
Good idea, done!

Reply:
It does not work in qesap regression context. `is_azure` is internally implemented with more tests than `get_var('PUBLIC_CLOUD_PROVIDER') eq 'AZURE'`  . One of them is not passing in qesap regression so the `is_azure` return `False`

Reply:
Reversed, because `is_azure` also checks for `is_public_cloud` which is not true in this context, as noticed by @mpagot

Reply:
Still could've used `check_var()` though:
```
if (check_var('QESAPDEPLOY_FENCING', 'native') && check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE')) {
```

Reply:
Now that we have https://gitlab.suse.de/qa-css/openqa_ha_sap/-/merge_requests/661 we can go back on it